lib/jekyll-import/importers/mt.rb

```diff
@@ -58,15 +65,25 @@ def self.specify_options(c)
       def self.process(options)
         options  = default_options.merge(options)
 
+        engine   = options.fetch('engine')
         dbname   = options.fetch('dbname')
         user     = options.fetch('user')
         pass     = options.fetch('password', "")
         host     = options.fetch('host', "localhost")
+        port     = options.fetch('port')
         comments = options.fetch('comments')
 
         posts_name_by_id = { } if comments
 
-        db = Sequel.mysql(dbname, :user => user, :password => pass, :host => host)
+        if engine == 'mysql'
+          db = Sequel.mysql(dbname, :user => user, :password => pass, :host => host, :port => port )
+        elsif engine == 'postgres'
+          db = Sequel.postgres(dbname, :user => user, :password => pass, :host => host, :port => port )
+        else
+          abort("unsupported --engine value supplied. Only 'mysql' and 'postgres' are supported")
```

Comment:
Might be good to print out the value we got in quotes so that they can map that to the command they gave.